lib/please_cli/please_cli/decision_task.py

```diff
@@ -511,6 +511,8 @@ def cmd(ctx,
         deploy_tasks = {}
         for index, (project, project_requires, deploy_target, deploy_options) in \
                 enumerate(sorted(projects_to_deploy, key=lambda x: x[0])):
+            if not deploy_options.get('enable'):
```

Comment:
@garbas The deployment will be disabled if the option is missing. I think we should either throw an exception if it's missing, or allow the deployment by default